src/store/index.js

```diff
@@ -45,6 +47,8 @@ async function fetchUser(commit) {
 async function retryFetchUser(commit, retries = 120) {
   if (!(await fetchUser(commit)) && retries > 0) {
     setTimeout(() => retryFetchUser(commit, retries - 1), 1000);
+  } else if (!userFound) {
```

Comment:
Not sure if this is a better approach here, I am not used to front end or Vue! Let me know